### PR TITLE
build: update service to Keptn 0.9.2

### LINKED ISSUE #30: Make argo-service compatible with Keptn 0.9.2
As mentioned in our [public slack](https://keptn.slack.com/archives/CGS5TA283/p1635543694018800) by @grabnerandi , the latest version of argo-service is compatible with Keptn 0.9.2, but it's not reflected in the README / releases page.

# Definition of Done

- [ ] Everything Keptn 0.8.x related is updated to Keptn 0.9.x
- [ ] README is updated to reflect that argo-service is compatible with Keptn 0.9.2
- [ ] New release

### PR DESCRIPTION
**Pull request by TannerGabriel**
Update the Argo-Service to be compatible with Keptn version 0.9.0-0.9.2.

## Fixes
#30

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,8 +3,31 @@ module github.com/keptn-contrib/argo-service
...
+	github.com/go-openapi/analysis v0.21.1 // indirect
+	github.com/go-openapi/runtime v0.21.0 // indirect
+	github.com/go-openapi/strfmt v0.21.1 // indirect
+	github.com/go-stack/stack v1.8.1 // indirect
+	github.com/google/uuid v1.3.0
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
-	github.com/keptn/go-utils v0.8.0
+	github.com/keptn/go-utils v0.11.0
```
Comment (christian-kreuzberger-dtx) [inline go.mod:19]: Should we not use go-utils v0.9.x here?

Comment (TannerGabriel) [reply go.mod:19]: Yes, if we only use this for Keptn v0.9.x that makes sense. Another possibility would be to try out this repository version with Keptn version 0.10.x and 0.11.x and maybe use them for all, which would mean that we don't need three different releases.  

Let me know how you would like to proceed here.

Comment (christian-kreuzberger-dtx) [reply go.mod:19]: I'll accept it as it is, shouldn't be anything in there that stops us :)

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM

**Pull request merged**